modules/core/src/utils/typed-array-manager.js

```diff
@@ -19,13 +19,12 @@ export class TypedArrayManager {
       }
     }
 
-    const newArray = this._allocate(Type, newSize);
+    const newArray = this._allocate(Type, newSize, initialize);
 
     if (typedArray && copy) {
       newArray.set(typedArray);
-    } else {
-      // Hack - viewing a buffer with a different type may create NaNs
-      // which crashes the Attribute validation
+    } else if (!initialize) {
+      // Hack - always initialize the first 4 elements. NaNs crashe the Attribute validation
```

Comment:
Nit: Seems weird to have a workaround for an attribute bug in a general typed-array-manager (gives this base component knowledge about a using component which is generally not desirable). Maybe this fix was pushed too far down in the stack?

Reply:
Agreed, hence the "hack" flag. We need this patch for production; I will open a separate PR to address that.